idunn/places/base.py

```diff
@@ -272,6 +272,15 @@ def get_appointment_url(self):
     def get_quotation_request_url(self):
         return None
 
+    def has_click_and_collect(self):
+        return False
+
+    def has_delivery(self):
+        return self.properties.get("delivery") == "yes"
+
+    def has_takeaway(self):
+        return self.properties.get("takeaway") == "yes"
```

Comment:
Any reason not to include `takeaway=only` which has a significant usage according to https://taginfo.openstreetmap.org/keys/takeaway#values ?

```suggestion
    def has_takeaway(self):
        return self.properties.get("takeaway") in ("yes", "only")
```